src/main/cpp/t02_new.cpp

```diff
@@ -25,10 +25,19 @@
 
 #include "t02_new.h"
 #include <iostream>
+#include <set>
 
 
 using namespace std;
 
 int t02_new() {
-    
+    set <int> s;
+    int n,x;
+    for (int i = 0; i < n; i++) {
+        cin >> x;
+        if (s.find(x)) cout << "YES";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Suspicious condition. The result of find() is an iterator, but it is not properly checked.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4763739)